openstack/compute/v2/extensions/attachinterfaces/results.go

```diff
@@ -0,0 +1,43 @@
+package attachinterfaces
+
+import (
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// Interface represents an interface.
+type FixedID struct {
+	SubnetID  string `json:"subnet_id"`
+	IpAddress string `json:"ip_address"`
+}
+
+// Interface represents an interface.
```

Comment:
How about:

// Interface represents a network interface on an instance.

Just so it's not confused as a golang interface.